examples/demo_astradb.ipynb

```diff
@@ -0,0 +1,323 @@
+   "metadata": {},
+   "source": [
+    "### Requirements"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First, install the required dependencies\n",
+    "!pip install --quiet llama-index llama-parse astrapy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import openai\n",
+    "\n",
+    "from getpass import getpass\n",
+    "\n",
+    "# Get all required API keys and parameters\n",
+    "llama_cloud_api_key = getpass(\"Enter your Llama Index Cloud API Key: \")\n",
+    "api_endpoint = input(\"Enter you Astra DB API Endpoint: \")\n",
+    "token = getpass(\"Enter your Astra DB Token: \")\n",
+    "openai_api_key = getpass(\"Enter your OpenAI API Key: \")\n",
+    "\n",
+    "os.environ[\"LLAMA_CLOUD_API_KEY\"] = llama_cloud_api_key\n",
+    "openai.api_key = openai_api_key"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# llama-parse is async-first, running the sync code in a notebook requires the use of nest_asyncio\n",
+    "import nest_asyncio\n",
+    "\n",
+    "nest_asyncio.apply()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using llama-parse to parse a PDF"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Download complete.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Grab a PDF from Arxiv for indexing\n",
+    "import requests \n",
+    "\n",
+    "# The URL of the file you want to download\n",
+    "url = \"https://arxiv.org/pdf/1706.03762.pdf\"\n",
+    "# The local path where you want to save the file\n",
+    "file_path = \"./attention.pdf\"\n",
+    "\n",
+    "# Perform the HTTP request\n",
+    "response = requests.get(url)\n",
+    "\n",
+    "# Check if the request was successful\n",
+    "if response.status_code == 200:\n",
+    "    # Open the file in binary write mode and save the content\n",
+    "    with open(file_path, \"wb\") as file:\n",
+    "        file.write(response.content)\n",
+    "    print(\"Download complete.\")\n",
+    "else:\n",
+    "    print(\"Error downloading the file.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Started parsing the file under job_id ba637beb-157c-4082-9f83-621d97fec6f8\n"
+     ]
+    }
+   ],
+   "source": [
+    "from llama_parse import LlamaParse\n",
+    "\n",
+    "documents = LlamaParse(result_type=\"text\").load_data(\"./attention.pdf\")"
```

Comment:
do you want to change this to use the `file_path` variable from above?

Reply:
Good call @epinzur . Updated now!